### ISSUE TITLE
Enclosures seem broken

### ISSUE THREAD

**Issue by ralsina** [getnikola/nikola#2189]
This is supposed to create an enclosure in the RSS feed, and doesn't:

```
.. title: The R-Podcast Episode 14: Tips and Tricks for using R-Markdown
.. slug: the-r-podcast-episode-14-tips-and-tricks-for-using-r-markdown
.. date: 2015-11-18 14:07:21
.. tags: Podcast
.. description: 
.. wp-status: publish
.. enclosure: https://archive.org/download/trp_audio_014/rpodcast014.mp3

I just did a fresh nikola deploy and the feed still isn't showing this episode's mp3 file.  Here's the default feed in case it helps: http://dev.r-podcast.org/rss.xml .  
```

**Comment by ralsina** [comment]
Fixed by 497d83eb20accc102af673e8955138d3e6a79d2c

**Comment by ralsina** [comment]
Fixed.
